insights/specs/default.py

```diff
@@ -143,7 +143,7 @@ class DefaultSpecs(Specs):
     cmdline = simple_file("/proc/cmdline")
     corosync = simple_file("/etc/sysconfig/corosync")
 
-    @datasource(HostContext, [IsRhel7, IsRhel8])
+    @datasource(HostContext, [IsRhel7, IsRhel8, IsRhel9])
```

Comment:
Sorry for the delay, and Just FYI, the author of this datasource @huali027, now is checking if this command works the same in RHEL 8 and RHEL 9. And will get back to you.

Reply:
That's okay thank you for letting me know, I just checked the package versions and saw that corosync in r9 and r8 were based on the same upstream version. But actually checking on a setup system would be a much better verification.

Reply:
@xiangce @ryan-blakley good to me, I found the commands on RHEL9 are the same with RHEL8.